requirements/edx/base.txt

```diff
@@ -549,7 +550,7 @@ event-tracking==2.1.0
     #   -r requirements/edx/base.in
     #   edx-proctoring
     #   edx-search
-fastavro==1.4.12
+fastavro==1.5.1
```

Comment:
No breaking change mentioned in the [CHANGELOG](https://github.com/fastavro/fastavro/blob/master/ChangeLog#L4).